app/_landing_pages/gateway/openid-connect.yaml

```
@@ -0,0 +1,293 @@
+    - gateway
+  works_on:
+  - on-prem
+  - konnect
+  tags:
+  - openid-connect
+  - authentication
+  - authorization
+rows:
+  - header:
+      type: h1
+      text: "OpenID Connect with {{site.base_gateway}}"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    OpenID Connect (OIDC) is a standard built on top of OAuth and JWT (JSON Web Token). 
+                    It provides authentication and authorization, letting you connect {{site.base_gateway}} to an identity provider (IdP),
+                    where the system you interact with can determine who you are and give you access to the correct resources.
+
+                    {{site.base_gateway}} provides an [OpenID Connect plugin](/plugins/openid-connect/) with support for a large variety of auth flows and grants.
+                    With this plugin, you can leverage an IdP for both authentication and authorization without having to manage consumers or groups in {{site.base_gateway}}, 
+                    giving you the ability to provide secure developer access without additional operational overhead and risk. 
+
+                    The OIDC plugin also includes session management and support for the Financial-Grade API (FAPI) standard.
+
+
+  - header:
+      type: h2
+      text: "Get started with OIDC"
+    columns:
+      - blocks:
+        - type: card
+          config:
+            title: Try the authorization code flow
+            description: |
+              The authorization code flow is one of the most commonly used workflows with OpenID Connect. 
+              
+              Set it up with Keycloak as a sample identity provider (IdP) to test out the OIDC plugin.
+            icon: /assets/icons/graduation.svg
+            cta:
+              text: Configure your first OIDC implementation &rarr;
+              url: /how-to/configure-oidc-with-auth-code-flow/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Review supported authentication flows and grants
+            description: Learn about the authentication options that the OIDC plugin supports, and choose the one that suits your use case.
+            icon: /assets/icons/lock.svg
+            cta:
+              text: Learn about supported flows and grants &rarr;
+              url: "#authentication-flows-and-grants"
+              align: end
+      - blocks:
+        - type: plugin
+          config:
+            slug: openid-connect
+
+  - header:
+      type: h2
+      text: "Authentication flows and grants"
+    columns:
+      - blocks:
+          - type: table
+            config:
+              columns:
+                - title: Authentication flow or grant
+                  key: auth
+                - title: Description
+                  key: description
+                - title: Examples in every supported tool
```

```suggestion
                - title: Plugin example configuration
```
The original title was a bit confusing